plugins/sql-language/src/main/java/org/polypheny/db/sql/language/SqlIntervalLiteral.java

```diff
@@ -17,15 +17,17 @@
 package org.polypheny.db.sql.language;
 
 
+import static org.polypheny.db.sql.language.SqlIntervalQualifier.intervalString;
```

Comment:
Why a static import and not SqlIntervalQualifier.intervalString(..)

Reply:
Intellj, adjusted...